KeysExample.swift

```diff
@@ -0,0 +1,2 @@
+
```

Comment:
Leading Whitespace Violation (Medium Severity): File shouldn't start with whitespace: currently starts with 1 whitespace characters<br>Name Format Violation (Medium Severity): Variable name should be between 3 and 40 characters in length: 'x'